### DISCUSSION TITLE
When the new App store server API implementation coming into this package?

### DISCUSSION THREAD

**Discussion by Abhishek-kumar-appfoster** [imdhemy/appstore-iap#188] [Q&A]
Hi, there @imdhemy 
Will the new app store server api( subscriptions status and getTransationHisttory) is going to be implemented in this package?

**Comment by imdhemy** [comment]
Hi @Abhishek-kumar-appfoster 
Could you please share documentation links?

**Reply by Abhishek-kumar-appfoster** [reply]
Hi @imdhemy 
This is new server api that apple has announced ig back in WWDC21 
https://developer.apple.com/documentation/appstoreserverapi.

**Reply by Tosh79** [reply]
Hi @imdhemy ,

do you have any plans integrating the server api to this really great package? App Store Receipts are tagged "deprecated":

https://developer.apple.com/documentation/appstorereceipts?changes=latest_minor

Cheers!

**Reply by imdhemy** [reply]
Hi @Tosh79 
Yes, I do. I need to find time for it. I'm currently busy with other projects. PRs are welcomed.
